packages/dx-react-grid/docs/reference/searching-state.md

```diff
@@ -0,0 +1,34 @@
+# SearchingState Plugin Reference
+
+A plugin that manages the search value state.
+
+## User Reference
+
+### Dependencies
+
+none
+
+### Properties
+
+Name | Type | Default | Description
+-----|------|---------|------------
+searchValue? | string |  | Specifies the applied search value;
+defaultSearchValue? | string |  | Specifies the search value initially applied in the uncontrolled mode.
+onSearchValueChange? | (searchValue: string) => void | | Handles search value changes.
+
+## Plugin Developer Reference
+
+### Imports
+
+Name | Plugin | Type | Description
+-----|--------|------|------------
+filterExpr| Getter | [IntegratedFiltering.FilterExpr](integrated-filtering.md/#integratedfilteringfilterexpr) | The applied filter expressions.
+columns | Getter | Array&lt;[Column](grid.md#column)&gt; | Grid columns.
+
+### Exports
+
+Name | Plugin | Type | Description
+-----|--------|------|------------
+searchValue | Getter | string | The applied search value
+filterExpr| Getter | [IntegratedFiltering.FilterExpr](integrated-filtering.md/#integratedfilteringfilterexpr) | The applied filter expressions.
```

Comment:
Can we find a better name? Don't use abbreviations. The simplest one: `filterExpression`.

Reply:
I think filterExpression**s** is better. But abbreviations like RegExp is a common case. The other example: https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#keyExpr

Reply:
@SergeyAlexeev What do you think about this?

Reply:
I'd prefer `filterExpr`. I think it's a kind of sustainable expression))

Reply:
@viterobk

Reply:
Just want to note that we didn't use abbreviations in API before. Not sure that we should do it now.

Reply:
Recently, we discussed the same option in DevExtreme DataGrid. We use expr in options that define which field (or expression) holds the required value (e.g. keyExpr, selectedExpr, valueExpr). For filter expression we decided to use "Expression". Thus, if we will also follow this rule, it's better to use filterExpression.